integration/ipv6/ipv6.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+source "/etc/os-release" || "source /usr/lib/os-release"
+image="debian"
+payload="tail -f /dev/null"
+first_container_name="test1"
+second_container_name="test2"
+docker_configuration_path="/etc/docker"
+docker_configuration_file=$docker_configuration_path/daemon.json
+
+setup() {
+	clean_env
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+	cat << EOF | sudo tee "${docker_configuration_file}"
+	{
+	"ipv6": true,
+	"fixed-cidr-v6": "2001:db8:1::/64"
+	}
+EOF
+	# Restart docker
+	sudo systemctl restart docker
+}
+
+@test "test ipv6" {
+	# Run first container
+	docker run -d --name "${first_container_name}" --runtime kata-runtime "${image}" sh -c "${payload}"
+
+	# Run second container
+	docker run -d --name "${second_container_name}" --runtime kata-runtime "${image}" sh -c "${payload}"
+
+	# Check ipv6
+	check_ipv6=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.GlobalIPv6Address}}{{end}}' "${second_container_name}")
+	echo "${check_ipv6}" | grep "2001:db8:1"
+
+	# Ping containers
+	run docker exec "${first_container_name}" sh -c "ping -c 6 ${check_ipv6}"
+	[ "$status" -eq 0 ]
+}
+
+teardown() {
+	clean_env
+	rm -rf $docker_configuration_path
```

Comment:
this would delete other configuration options we have in the file. I think the best would be to 
1. backup `/etc/daemon.json`, then modify it at `setup`,
2. run the test and 
3. restore that backup in the `teardown`.

Reply:
@chavafg changes applied